src/core/mount.c

```diff
@@ -11,7 +11,6 @@
 #include "dbus-mount.h"
 #include "dbus-unit.h"
```

Comment:
this commit should probably be added to v242 still?

Reply:
Maybe the first three (refactoring, test, and cunescape removal)? I can submit separate PR.